modules/ztp-deploying-a-site.adoc

```diff
@@ -184,4 +184,9 @@ The `SiteConfig` CR creates the following CRs on the hub cluster:
 * `NMStateConfig` - One per node
 * `ExtraManifestsConfigMap` - Extra manifests. The extra manifests include workload partitioning, mountpoint hiding, sctp enablement, and more. To automatically merge the extra manifests into a single manifest per each `MachineConfigPool` role, which is named as `predefined-extra-manifests-<role>`, set the `.spec.clusters.mergeDefaultMachineConfigs` to `true` in the `SiteConfig.yaml` file.
 * `ManagedCluster`
-* `KlusterletAddonConfig`
\ No newline at end of file
+* `KlusterletAddonConfig`
+
+[role="_additional-resources"]
+.Additional resources
```

Comment:
The Additional resources section goes in the assembly file since we can not put `xrefs` in modules